src/lib/tvgSaver.cpp

```diff
@@ -117,6 +118,26 @@ Result Saver::save(std::unique_ptr<Paint> paint, const string& path) noexcept
 }
 
 
+Result Saver::save(std::unique_ptr<Paint> paint, const std::string& mimeType, uint8_t** buffer, uint32_t* size) noexcept
+{
+    //Already on saving an other resource.
```

Comment:
I think user could save multiple times.
ie) to multiple files and to memory.
for later enhancement, you can leave TODO: comment here.